observations/obs_converters/NCEP/prep_bufr/prep_bufr.rst

```diff
@@ -1,67 +1,315 @@
+   directory. For this example:
+
+   ::
+
+      &ncepobs_nml
+         year       = 2017,
+         month      = 4,
+         day        = 27,
+         tot_days   = 3,
+         max_num    = 800000,
+         select_obs = 0,
+         ObsBase = '../../prepbufr/data/prepout/temp_obs.',
+         daily_file = .false.,
+         lat1       = 15.0,
+         lat2       = 60.0,
+         lon1       = 200.0,
+         lon2       = 330.0
+         /
+
+   Choosing "select_obs = 0" will select all the observations in the
+   ASCII file. Set "ObsBase" to the directory you output the files from
+   during the last step. If you wish to choose specific observations
+   from the ASCII intermediate file or control other program behavior,
+   there are many namelist options documented on the
+   `create_real_obs <../../../observations/obs_converters/NCEP/ascii_to_obs/create_real_obs.html>`__
+   page.
+
+-  It is now time to build *ascii_to_obs* programs. Run the following:
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
+      ./quickbuild.sh
+
+-  Run the *create_real_obs* program to create the DART observation
+   sequence files:
+
+
+   ::
+
+      cd $DART_DIR/observations/obs_converters/NCEP/ascii_to_obs/work
+      ./create_real_obs
+
+-  The program *create_real_obs* will create observation sequence files
+   with one file for each six hour window. For a cycled experiment, the
+   typical approach is to put a single set of observations, associated
+   with a single analysis step, into a separate directory. For example,
+   within the ``output`` directory, we would create directories like
+   ``2017042700``, ``2017042706``, ``2017042712``, etc. for 6-hourly
+   cycling. Place the observation files in the appropriate directory to
+   match the contents in the files (e.g. *obs_seq2017042706*) and rename
+   as simply *obs_seq.out* (e.g. ``output/2017042706/obs_seq.out``).
+
+-  The observation types within the ds090 ``obs_seq2017042706`` file should look like:
+
+   ::
+
+     obs_sequence
+     obs_type_definitions
+          15
+          12 AIRCRAFT_U_WIND_COMPONENT
+          13 AIRCRAFT_V_WIND_COMPONENT
+          14 AIRCRAFT_TEMPERATURE
+          20 MARINE_SFC_U_WIND_COMPONENT
+          21 MARINE_SFC_V_WIND_COMPONENT
+          22 MARINE_SFC_TEMPERATURE
+          23 MARINE_SFC_SPECIFIC_HUMIDITY
+          25 LAND_SFC_U_WIND_COMPONENT
+          26 LAND_SFC_V_WIND_COMPONENT
+          27 LAND_SFC_TEMPERATURE
+          28 LAND_SFC_SPECIFIC_HUMIDITY
+          30 SAT_U_WIND_COMPONENT
+          31 SAT_V_WIND_COMPONENT
+          42 MARINE_SFC_ALTIMETER
+          43 LAND_SFC_ALTIMETER
+    num_copies:            1  num_qc:            1
+    num_obs:        19059  max_num_obs:        19059
+
+
+-  It is helpful to also run the
+   `wrf_dart_obs_preprocess <../../../models/wrf/WRF_DART_utilities/wrf_dart_obs_preprocess.html>`__
+   program, which can strip away observations not in the model domain,
```

Comment:
This seems pretty wrf specific, is this the general advice for prep_bufr observations when using other models.